src/support/utils.js

```diff
@@ -11,12 +11,18 @@
  */
 import { context as h2, h1 } from '@adobe/fetch';
 import { DELIVERY_TYPES } from '@adobe/spacecat-shared-data-access/src/models/site.js';
+import { getClient, CONTENT_TYPES } from 'spacecat-helix-content-sdk';
+import yaml from 'js-yaml';
 
 /* c8 ignore next 3 */
 export const { fetch } = process.env.HELIX_FETCH_FORCE_HTTP1
   ? h1()
   : h2();
 
+export const GOOGLE_DRIVE = 'google-drive';
```

Comment:
Maybe all these constants and methods related to the content client should go in an utils of their own?